### ISSUE TITLE
Dashboard "More" not rendering as dropdown

### ISSUE THREAD

**Issue by rigelifland** [dask/distributed#5368]
Windows 10, MacOS Big Sur
Python 3.7 (Virtualenv built from a miniconda install of python)
distributed==2021.09.1

I was able to reproduce this on Windows and MacOS - at least partially. On MacOS I installed it into a fresh virtualenv and it correctly rendered. I went back to my original environment and it too started rendering correctly. On Windows, in an offline environment, I have been unable to get it to render correctly. I'm assuming that there's some stylesheet that isn't being loaded but I don't see any failed requests when I use browser dev-tools to inspect traffic.

<img width="1440" alt="Screen Shot 2021-09-29 at 8 02 16 AM" src="https://user-images.githubusercontent.com/21069069/135295648-f13e9da3-383f-4b12-bcb6-8bf288b61b7f.png">

**Comment by jrbourbeau** [comment]
Thanks for raising an issue @rigelifland! cc @jacobtomlinson if you have any thoughts on the topic

**Comment by jacobtomlinson** [comment]
If things are working correctly on a fresh install I'm not sure how I can go about reproducing this. My first thought goes to things like clearing your browser cache.

**Comment by rigelifland** [comment]
I've tried a number of times to reproduce this (in a variety of environments) and failed. I'm not sure why I was able to get the same issue on two totally separate systems, but fresh installs fixed the problem both times, and I haven't been able to determine if it's a cache issue or some odd package incompatibility. 

I'd recommend closing this until someone bumps into the problem again and can consistently reproduce it.
